test/core/client_channel/lb_policy/xds_override_host_test.cc

```diff
@@ -199,46 +206,35 @@ TEST_F(XdsOverrideHostTest, SubchannelConnectingIsQueued) {
   ASSERT_NE(subchannel, nullptr);
   subchannel->SetConnectivityState(GRPC_CHANNEL_IDLE);
   ExpectReresolutionRequest();
+  picker = ExpectState(GRPC_CHANNEL_READY);
+  ExpectPickQueued(picker.get(), pick_arg);
```

Comment:
Now that you've added this, I think this test is actually covering both the IDLE and CONNECTING cases, so we don't need a separate test for IDLE below.

Reply:
Gone now!